src/status_im/chat/subs.cljs

```diff
@@ -296,7 +296,8 @@
 
          (and command
               (= (count real-args) 1)
-              (input-model/text-ends-with-space? input))
+              ;; TODO(goranjovic) - why do we need this? it will never be true if the string was just trimmed
+              #_(input-model/text-ends-with-space? input))
```

Comment:
Remove it then?

Reply:
Left it for the review just in case it's doing something super critical. Everything works fine without it, so apparently not.